### PR TITLE
NODE-2523 Swagger large-significand-format

### PR DESCRIPTION
**Pull request by darksyd94**
Draft.
Added new examples with usage of components/examples.

### CODE REVIEW

**Review by vsuharnikov — APPROVED** [review]

**Review by yvaskov — APPROVED** [review]

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 1]
```diff
@@ -3441,6 +3918,17 @@ components:
...
+      allOf:
+        - $ref: '#/components/schemas/BlockHeaderLSF'
+        - type: object
+          required:
+            - transactions
+          properties:
+            transactions:
+              type: array
+              items:
+                $ref: '#/components/schemas/Transaction'
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml]: `TransactionLSF` should be here

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 2]
```diff
@@ -3227,6 +3492,24 @@ components:
...
+    AssetDistributionLSF:
+      required:
+        - hasNext
+        - last
+        - values
+      type: object
+      properties:
+        hasNext:
+          type: boolean
+        last:
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml]: `lastItem` should be here and in `AssetDistribution` description

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 3]
```diff
@@ -3227,6 +3492,24 @@ components:
...
+          $ref: '#/components/schemas/Address'
+        values:
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml]: `items` should be here and in `AssetDistribution` description

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 4]
```diff
@@ -2275,6 +2474,9 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AssetDistribution'
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml]: Response of `/assets/{assetId}/distribution` contains only map of assetId <-> balance

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 5]
```diff
@@ -2275,6 +2474,9 @@ paths:
...
+            application/json;large-significand-format=string:
+              schema:
+                $ref: '#/components/schemas/AssetDistributionLSF'
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml]: Response of `/assets/{assetId}/distribution` contains only map of assetId <-> balance

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 6]
```diff
@@ -3244,6 +3527,23 @@ components:
           $ref: '#/components/schemas/AssetDescription'
         decimals:
           $ref: '#/components/schemas/AssetDecimals'
+    AssetDetailsBaseLSF:
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml:3573]: Description of `quantity` field is missed here

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 7]
```diff
@@ -3227,6 +3492,24 @@ components:
...
+        last:
+          $ref: '#/components/schemas/Address'
+        values:
+          additionalProperties:
+            type: string
+          description: map of assetId <-> balance
+          example:
+            2eEUvypDSivnzPiLrbYEW39SM8yMZ1aq4eJuiKfs4sEY: "15"
+            3PPqZ623dAfbmxmnpTjwV6yD5GA5s3PJiUG: "25"
     AssetDetailsBase:
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml:3552]: Description of `quantity` field is missed here

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 8]
```diff
@@ -2316,6 +2518,9 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AssetDistribution'
+            application/json;large-significand-format=string:
+              schema:
+                $ref: '#/components/schemas/AssetDistributionLSF'
   '/assets/nft/{address}/limit/{limit}':
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml:2549]: Response of this method with large-signicand-format is missed

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 9]
```diff
@@ -2590,8 +2795,8 @@ paths:
                   format: int64
                 description: map of address <-> balance
                 example:
-                  addr1: 0
-                  addr2: 100
+                  addr1: "0"
+                  addr2: "100"
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml:2838]: There is no description of `/debug/state` response with large-significand-format

File: node/src/main/resources/swagger-ui/openapi.yaml [hunk 10]
```diff
@@ -2920,6 +3138,15 @@ paths:
...
                   addr1: 0
                   addr2: 100
+            application/json;large-significand-format=string:
+              schema:
+                type: object
+                additionalProperties:
+                  type: string
+                description: map of address <-> balance
+                example:
+                  addr1: 0
```
Comment (ivan-mashonskiy) [inline node/src/main/resources/swagger-ui/openapi.yaml]: Example here should contain stringified numbers

**Comment by ivan-mashonskiy** [comment]
Description of `/assets/balance/{address}/{assetId}` response with large-significand-format is missed

**Review by ivan-mashonskiy — APPROVED** [review]

**Review by yvaskov — APPROVED** [review]

**Review by phearnot — APPROVED** [review]

**Pull request merged**